### PR TITLE
Tnx

### PR DESCRIPTION
**Pull request by gptucci**
:pencil2: **Blog Url**: 

:scroll: **Feed Url**: 

By sending this pull request to add my blog I verify that I adhere to the Planet Xamarin blog guidelines (please check each item that applies):

- [x] I have a valid blog & RSS URL, both using HTTPS :lock: with a valid certificate
- [x] I host NO malicious or offensive content on the blog (including photos, swearing, etc.)
- [x] My blog is active with at least 3 Xamarin related blog posts in the last 6 months
- [x] In addition to the previous guideline; it should be apperant that the blog is active for a longer period of time and the posts on it are not fabricated in a small amount of time to comply to the '3 posts in last 6 months' guideline.
- [ ] I have applied a filter (Optional: if blog is only about Xamarin or you use `xamarin` in the title or categories in your blog posts)
- [x] I have added my Author class to the csproj file
- [x] I have implemented `IAmACommunityMember` or one of the other derivatives
- [x] I understand, if I delete my blog, I will be deleted from Planet Xamarin
- [x] I understand, my blog may be removed at any time if any of these guidelines are broken.
- [x] The submitted blog is not owned by a company which sole purpose it is to advertise a product of productline.

Only the collaborators of this repository decide whether a blog is eligible or not. They will determine if the rules are followed and if the quality of the content is high enough. 

Opening another PR, after the first one is closed in a small period of time will not be reevaluated. If a PR is closed, the issues with your blog are too significant to be fixed within a matter of hours.

### CODE REVIEW

**Comment by jfversluis** [comment]
You have a build error: `'GiampaoloTucci' does not implement interface member 'IAmACommunityMember.GravatarHash'`

File: src/Firehose.Web/Authors/GiampaoloTucci.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Collections.Generic;
+using Firehose.Web.Infrastructure;
+
+namespace Firehose.Web.Authors
+{
+    public class GiampaoloTucci : IAmACommunityMember, IAmAPodcast
+    {
+        public Uri WebSite => new Uri("https:/www.informaticapressapochista.com/it");
```
Comment (Cheesebaron) [inline src/Firehose.Web/Authors/GiampaoloTucci.cs]: This is an invalid url

File: src/Firehose.Web/Authors/GiampaoloTucci.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        public string FirstName => "Giampaolo";
+        public string LastName => "Tucci";
+        public string StateOrRegion => "Italia";
+        public string EmailAddress => "g.tucci@informaticapressapochista.com";
+        public string ShortBioOrTagLine => "L'IT come non l'avete mai letta";
+        public string GravatarHash => "26850a95739983a8280262b0044fe3e5";
```
Comment (Cheesebaron) [inline src/Firehose.Web/Authors/GiampaoloTucci.cs]: Your gravatar hash returns an empty image

File: src/Firehose.Web/Authors/GiampaoloTucci.cs [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        
+        public IEnumerable<Uri> FeedUris
+        {
+            get { yield return new Uri("https://www.informaticapressapochista.com/it/?format=feed&type=rss"); }
```
Comment (Cheesebaron) [inline src/Firehose.Web/Authors/GiampaoloTucci.cs]: Your feed doesn't seem to be parsable by BlogMonster, which we use to parse feeds. Sorry, but I don't think we are going to change this right now.

Comment (Cheesebaron) [reply src/Firehose.Web/Authors/GiampaoloTucci.cs]: Nevermind, it is actually the filtering that filters away all your posts. Right now it returns an empty feed, which we don't accept.

**Comment by gptucci** [comment]
Tnx
Today I'll check

**Comment by gptucci** [comment]
Solved ?! Could you check now ?? I've created a xamarin category. Tnx

File: src/Firehose.Web/Authors/GiampaoloTucci.cs (hunk 3 above)
Comment (gptucci) [reply src/Firehose.Web/Authors/GiampaoloTucci.cs]: I've modified my blog. I suppose that now works

**Comment by Cheesebaron** [comment]
Your feed still doesn't contain any items with either Xamarin in title or Xamarin as category.

**Comment by gptucci** [comment]
And now ?? Really tnx

**Comment by gptucci** [comment]
I've modified the filter
Now works ??
tnx

**Comment by gptucci** [comment]
feed of JunianTriajianto is in error !

File: src/Firehose.Web/Authors/GiampaoloTucci.cs [hunk 4]
```diff
@@ -18,7 +18,14 @@ public IEnumerable<Uri> FeedUris
         {
             get { yield return new Uri("https://www.informaticapressapochista.com/it/?format=feed&type=rss"); }
         }
-
+        
+        public bool Filter(SyndicationItem item)
+        {    
+            if ((item.Links?.Any(c => c.Uri.ToString().ToLowerInvariant().Equals("xamarin")) ?? false))
```
Comment (Cheesebaron) [inline src/Firehose.Web/Authors/GiampaoloTucci.cs]: If Uri contains xamarin, wouldn't the title or category already contain it?

**Comment by gptucci** [comment]
Yes, but I'm using Joomla, and without it I've verified that... does not work ! All posts are assigned to Xamarin category. Tnx

**Comment by gptucci** [comment]
Now work w/ filter - tnx

**Comment by Cheesebaron** [comment]
Your feed still only uses General as category. Are you sure this is correct?

**Comment by gptucci** [comment]
Yes
I ll put xamarin word in title in evert related post
Tnx



Sent from my Samsung device


-------- Original message --------

**Pull request merged**